src/status_im/multiaccounts/create/core.cljs

```diff
@@ -138,13 +138,17 @@
 (fx/defn intro-step-forward
   {:events [:intro-wizard/step-forward-pressed]}
   [{:keys [db] :as cofx} {:keys [skip?] :as opts}]
-  (let  [{:keys [step first-time-setup? selected-storage-type processing?]} (:intro-wizard db)]
+  (let  [{:keys [step first-time-setup? selected-storage-type processing? weak-password?]} (:intro-wizard db)]
     (cond (confirm-failure? db)
           (on-confirm-failure cofx)
 
           (= step :generate-key)
           (init-key-generation cofx)
 
+          (and  (= step :create-code)
+                (= (get-in db [:intro-wizard :weak-password?]) true))
```

Comment:
i can see you destructured weak-password? in let why don't you use it here? and i'm not sure if we need a log so you can do smth like `(and  (= step :create-code) weak-password?) nil`

Reply:
@flexsurfer Just coding on not enough sleep.  Fixed in my latest commit.  I also replaced the logging message with a `nil` so should be ready for final review.